### PR TITLE
Add Directory Flag [-d]

### PR DESCRIPTION
**Pull request by bhrayhaw**
This shows the directory structure

### CODE REVIEW

File: ls/ls.go [hunk 1]
```diff
@@ -56,3 +60,19 @@ func isHiddenPath(path string, forceHidden bool) bool {
...
+func (l *LS) ShowDirStructure() error {
+    file, err := os.Stat(l.Dir)
+	if err != nil {
+        return err
+    }
+ 
+	if file.IsDir() {
+		fmt.Fprintln(l.StdOut, file.Name()+"/")
+	}else{
+		fmt.Println(file.Name())
```
Comment (profclems) [inline ls/ls.go]: use `fmt.Fprintln(l.StdOut, file.Name())`

**Review by profclems — CHANGES_REQUESTED** [review]
LGTM... Just some minor comments to fix. Also run `make fix` before committing

File: ls/ls.go (hunk 1 above)
Comment (bhrayhaw) [reply ls/ls.go]: Done correcting

**Review by Bostigger — APPROVED** [review]
Good job!

File: ls/ls.go (hunk 1 above)
Comment (KingKhay) [reply ls/ls.go:73]: LGTM. Keep up the good work

**Review by KingKhay — COMMENTED** [review]
LGTM. Keep up the good work!!

**Review by KingKhay — APPROVED** [review]

**Review by profclems — APPROVED** [review]

**Pull request merged**